src/SFA.DAS.AANHub.Api.UnitTests/Controllers/EmployersControllerTests.cs

```diff
@@ -71,5 +71,101 @@ public EmployersControllerTests()
 
             result.As<BadRequestObjectResult>().StatusCode.Should().Be(StatusCodes.Status400BadRequest);
         }
+
+        [Test]
+        [AutoMoqData]
+        public async Task GetEmployer_InvokesQueryHandler(
+            ValidatedResponse<GetEmployerMemberResult> handlerResult)
+        {
+            _mediator.Setup(m => m.Send(It.IsAny<GetEmployerMemberQuery>(), It.IsAny<CancellationToken>())).ReturnsAsync(handlerResult);
+
+            var userRef = new Guid();
+            var response = await _controller.GetEmployer(userRef);
+            response.Should().NotBeNull();
+
+            var result = response as OkObjectResult;
+            Assert.AreEqual(StatusCodes.Status200OK, result!.StatusCode);
+
+            var queryResult = result.Value as GetEmployerMemberResult;
+            queryResult.Should().BeEquivalentTo(handlerResult.Result);
+
+            _mediator.Verify(m => m.Send(It.IsAny<GetEmployerMemberQuery>(), It.IsAny<CancellationToken>()));
```

Comment:
Could verify that the passed in UserRef is set in the query by using ~ `It.Is<GetEmployerMemberQuery>(q => q.UserRef == userRef)`

Reply:
Test has been updated with suggestion